modules/wri_person/config/install/core.entity_form_display.node.person.default.yml

```diff
@@ -99,6 +106,40 @@ third_party_settings:
         formatter: open
         description: ''
         required_fields: true
+    group_publishing:
```

Comment:
This publishing group is WRI specific too, so might give you grief during site install. If not, you can leave it.